extension-definition-specifications/incident-ef7/Incident Extension Suite.adoc

```diff
@@ -1118,6 +1123,12 @@ Using these embedded relationships ensures that an incomplete sequence cannot be
 |[stixtype]#{location_url}[location]#
 |The task occurred at a specific location.
 
+
+|[stixtype]#<<task,task>>#
+|[stixrelationship]#performed-by#
+|[stixtype]#{identity_url}[identity]#
+|The task was performed by the identity.
+
```

Comment:
If you have a "performed-by" relationship, should the "performed" reverse relationship be removed (see new line number 1159)?

Reply:
I wanted to make them consistent.  so both are preformed and reverse relationships